tests/unit/initializers/engines-test.js

```diff
@@ -15,8 +14,6 @@ module('Unit | Initializer | engines', {
 
 // Replace this with your real tests.
 test('it works', function(assert) {
-  EnginesInitializer.initialize(application);
-
   // you would normally confirm the results of the initializer here
   assert.ok(true);
 });
```

Comment:
This test file doesn't do anything productive anymore. Do you have plans for it or should it be removed?